src/third_party/geospatial/geospatial_aggr.h

```diff
@@ -16,10 +17,98 @@
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
 
+#include <iostream>
```

Comment:
remove STL stuff and #include <inttypes.h> - to print uint64 use TUInt64::GetStr()